habitat-baselines/habitat_baselines/config/instance_imagenav/ddppo_instance_imagenav_v3.yaml

```diff
@@ -0,0 +1,88 @@
+# @package _global_
```

Comment:
For my personal understanding, what is different compared to v2 besides the benchmark ?

Reply:
Just the benchmark, but we add `rl.policy.action_distribution_type: gaussian` for the action space.

Reply:
Would it be possible to use v2 in the default list and then add the benchmark / action type? I think that would be better so that when we change v2 the changes will affect v3 automagically.

Reply:
@jacobkrantz What do you think about this?